src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    padding: 23px 0;
+
+    font-size: $font-size--title;
+    font-weight: $font-weight--title;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $text-color--main;
+    transition: color 0.5s;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+
+}
+
+.is-active {
+  position: relative; //???
+  color: #00acdc;
+
+  &::before {   //??
+    content: "";
+
+    position: absolute;//??
```

Comment:
What does this comments mean? 🙂

Reply:
Sorry))
That was my drafts, while I was chosen what is better to apply)

Reply:
deleted